L1Trigger/Phase2L1ParticleFlow/plugins/L1MHtPFProducer.cc

```diff
@@ -0,0 +1,98 @@
+#include <vector>
+#include <numeric>
+
+////////////////////
+//// FRAMEWORK HEADERS
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/L1TParticleFlow/interface/PFCandidate.h"
+#include "DataFormats/L1TParticleFlow/interface/PFJet.h"
+
+#include "DataFormats/L1Trigger/interface/EtSum.h"
+#include "DataFormats/Math/interface/LorentzVector.h"
+
+// bitwise emulation headers
+#include "L1Trigger/Phase2L1ParticleFlow/src/newfirmware/jetmet/L1PFHtEmulator.h"
+
+class L1MhtPfProducer : public edm::global::EDProducer<> {
+public:
+  explicit L1MhtPfProducer(const edm::ParameterSet&);
+  ~L1MhtPfProducer() override;
+
+private:
+  void produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetup& iSetup) const override;
+  edm::EDGetTokenT<std::vector<l1t::PFJet>> _jetsToken;
+  float _minJetPt;
```

Comment:
Following the CMSSW naming code rule, we should not start class member with "_".

https://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1